### PR TITLE
Replace references to IRC channel with Zulip chat

### PR DESCRIPTION
**Pull request by radusuciu**
IRC link is dead. Long live Zulip!

### CODE REVIEW

File: pages/get-involved.html [hunk 1]
```diff
@@ -37,7 +37,7 @@ <H3>{% trans "Start Developing Now" %}</H3>
       </li>
       <li>
         <p>
-          {% blocktrans with irc_channel_link="/irc/" %}When you're stuck, get help from our existing developers our IRC channel: #mixxx on <a href="{{ irc_channel_link }}">Freenode</a>{% endblocktrans %}
+          {% blocktrans with zulip_web_link="https://mixxx.zulipchat.com/" %}When you're stuck, get help from our existing developers our <a href="{{ zulip_web_link }}">Zulip chat</a>{% endblocktrans %}
```
Comment (Pegasus-RPG) [inline pages/get-involved.html]: existing developers **on** our

**Comment by Pegasus-RPG** [comment]
ok to test

File: pages/get-involved.html (hunk 1 above)
Comment (radusuciu) [reply pages/get-involved.html]: Fixed, thank you. I can squash commits if preferred.

**Comment by Be-ing** [comment]
LGTM thanks for catching this.

**Pull request merged**